src/data/injections.js

```diff
@@ -411,6 +411,21 @@ const AVAILABLE_INJECTIONS = [
       ],
     },
   },
+  {
+    id: "bug1644830",
+    platform: "desktop",
+    domain: "usps.com",
+    bug: "1644830",
+    contentScripts: {
+      matches: ["https://*.usps.com/*"],
```

Comment:
Note: I'm adding the wildcard here, as this probably is not limited to a specific subdomain. The registration is at `reg.usps.com`, but `missingmail.usps.com` also has affected checkboxes - so it's probably a save bet that there are other forms affected.